pkg/application/downloader.go

```diff
@@ -76,32 +76,24 @@ func (d downloader) DownloadWithTee(url string, path string) ([]byte, error) {
 }
 
 func (d downloader) DownloadWithCache(url string, path string, duration time.Duration) ([]byte, error) {
-	var data []byte
-	var useCache bool
-
-	// Check if cache file exists and is recent
-	if fileInfo, err := os.Stat(path); err == nil {
-		if time.Since(fileInfo.ModTime()) < duration {
-			// Cache is valid, read from it
-			data, err = os.ReadFile(path)
-			if err == nil {
-				useCache = true
+	data, err := d.Download(url)
```

Comment:
making downloading from git the first approach so that users always get newest version. If it fails we will default to cache